src/attendees/attendee.py

```diff
@@ -79,7 +89,17 @@ def set_attribute(self, key: Hashable, value: Any,
             TypeError if key is not hashable.
             ValueError if key is "fname", "lname", or "email"
         """
+        self.key = key
+        self.value = value
+        self.verifyFunc = verifyFunc
+
         raise NotImplementedError
+        
+        if type(key) is not Hashable:
+            raise TypeError("key is not hashable")
```

Comment:
This is not how you check if a variable is hashable. This should be automatically done by using a dicitonary to store the miscellaneous attributes.